src/utils.ts

```diff
@@ -119,4 +119,27 @@ function* zip(...args: any[]) {
 	}
 }
 
-export {hex2rgba, rgba2hex, hex2rgb, rgb2hex, rgb2hsv, hsv2rgb, zip, getMinZoom};
\ No newline at end of file
+function getOS() {
+  var userAgent = window.navigator.userAgent,
+      platform = window.navigator?.userAgentData?.platform || window.navigator.platform,
```

Comment:
Hmmm, my version of typescript (4.9.4) complains about not knowing about `window.navigator.userAgentData`. It works on my cases if I just use `window.navigator.platform`, even though it's deprecated. Does it work for yours?

Reply:
I actually encountered this issue too and forgot to move the change from my demo to the git repo. Using window.navigator.platform is what works for me too.